README.adoc

```diff
@@ -506,6 +506,19 @@ build:
     cd {{invocation_directory()}}; ./some_script_that_needs_to_be_run_from_here
 ```
 
+==== Justfile and Justfile Directory
+
+- `justfile()` - Retrieves the path to the current justfile
+
+- `justfile_directory()` - Retrieves the path to the parent directory of the current justfile
+
+For example, if you need to run a command relative to the current justfile, you could do the following:
```

Comment:
How about: "For example, to run a command relative to the location of the current justfile:"